regolith/sorters.py

```diff
@@ -4,12 +4,95 @@
 
 from regolith.dates import date_to_float
 
-doc_date_key = lambda x: date_to_float(x.get("year", 1970), x.get("month", "jan"))
-doc_date_key_high = lambda x: date_to_float(x.get("year", 1970), x.get("month", "dec"))
-ene_date_key = lambda x: date_to_float(x.get("end_year", 4242), x.get("end_month", "dec"))
-category_val = lambda x: x.get("category", "<uncategorized>")
-level_val = lambda x: x.get("level", "<no-level>")
-id_key = lambda x: x.get("_id", "")
+
+def doc_date_key(date):
+    """Convert a dict of Datetime object to float serialization of date info.
+
+    Parameters
+    ----------
+    date: dict
+        the document that is expected to contain date-like information in "year" and/or "month"
+
+    Returns
+    -------
+    the float serialization of the date information in the document
+    """
+    return date_to_float(date.get("year", 1970), date.get("month", "jan"))
+
+
+def doc_date_key_high(date):
+    """Convert a dict of highest Datetime object to float serialization of date info.
+
+    Parameters
+    ----------
+    date: dict
+        the document that is expected to contain date-like information in "year" and/or "month"
+
+    Returns
+    -------
+    float representing the date key.
+    """
+    return date_to_float(date.get("year", 1970), date.get("month", "dec"))
+
+
+def ene_date_key(date):
+    """Convert a dict of ene Datetime object to float serialization of date info.
+
+    Parameters
+    ----------
+    date: dict
+        the document that is expected to contain date-like information in "year" and/or "month"
+
+    Returns
+    -------
+    the float serialization of the date information in the document
+    """
+    return date_to_float(date.get("end_year", 4242), date.get("end_month", "dec"))
+
+
+def category_val(category):
+    """Convert the category of a dict of object into serialization of string of category info.
+
+    Parameters
+    ----------
+    category : dict
+        A dict of all corresponding categories for objects
+
+    Returns
+    -------
+    serialization of string representing the category.
```

Comment:
this is not a serialization, it is just returning the category item.  We probably don't need this function in fact.  Like elsewhere, we could just put the lambda inside the sorter, unless it is used in many places.  For now, just make changes to the docstring and let's go with that.